build/yamls/flow-visibility.yml

```diff
@@ -5009,6 +4916,26 @@ spec:
             name: clickhouse-configmap-volume
           - mountPath: /var/lib/clickhouse
             name: clickhouse-storage-volume
+        - env:
+          - name: CLICKHOUSE_USERNAME
+            valueFrom:
+              secretKeyRef:
+                key: username
+                name: clickhouse-secret
+          - name: CLICKHOUSE_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: password
+                name: clickhouse-secret
+          - name: DB_URL
```

Comment:
I think we don't need to store `DB_URL`, `TABLE_NAME`, and `MV_NAMES` in the environment variable after moving to a long-running Pod.

Reply:
I think the initial purpose to add these variables in the yaml file is in case of user would like to modify them, they can do it without rebuilt the monitor image? With a long-running Pod, we still use the monitor image.

Reply:
Got it, make sense to me.